conda/cli/conda_argparse.py

```diff
@@ -191,6 +192,12 @@ def _check_value(self, action, value):
         else:
             super()._check_value(action, value)
 
+    def parse_args(self, *args, **kwargs):
+        parsed_args = super().parse_args(*args, **kwargs)
+        for name, value in self.populate_args.items():
+            setattr(parsed_args, name, value)
+        return parsed_args
```

Comment:
we can just pass in `populate_args` here instead of modifying the `ArgumentParser` initialization:

```suggestion
    def parse_args(self, *args, populate_args=None, **kwargs):
        parsed_args = super().parse_args(*args, **kwargs)
        for name, value in populate_args.items():
            setattr(parsed_args, name, value)
        return parsed_args
```

I'm also thinking we should call it `override_args` instead

Reply:
Sounds good